source/Core/Threads/OperatingModes/DebugMenu.cpp

```diff
@@ -116,6 +121,20 @@ void showDebugMenu(void) {
       OLED::printNumber(hallEffectStrength, 6, FontStyle::SMALL);
     } break;
 #endif
+#ifdef MODEL_Pinecilv2
```

Comment:
If this is not defined it would be nicer to push the screen on one.

Reply:
You mean to jump to index 0? Yes, I think it already does this (although I didn't try on non-V2 yet).

Reply:
I'm thinking; in advance of this problem getting worse here.
Instead of having an ever increasing chain of #ifdef's;might be better to just skip in the switchcase handler to next entry and not have the ifdef mess at the bottom

Reply:
But doing the incrementing in the switch case is already too late, since the description message (i.e. "Tip R" or "ADC coe" has already been printed). then there would be one or more glitch frames every time we switch. The skipping would have to happen at the beginning of every frame draw, not in the middle.
Having the same bunch of ifdefs at the beginning of the frame draw is not any better than having it in the button press code.

One way to work around this would be to first draw the value, and only then the description string of the debug item but that would be a refactor that complicates the code.

Reply:
Yeah thats true.

Lets leave as-is like this, and ill note this function needs a cleanup some day in the future :)